lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -162,6 +162,71 @@ def test_staff_access(self):
         response = self.client.get(reverse('gitlogs'))
         self.assertTrue(response.status_code, 200)
 
+    def test_rename_user(self):
+
+        self._setstaff_login()
+        self.client.login(username=self.user.username, password='foo')
+
+        user1 = UserFactory.create(
+            username='test_rename_user_1',
+            email='test_rename_user_1@edx.org',
+            password='foo'
+        )
+        user2 = UserFactory.create(
+            username='test_rename_user_2',
+            email='test_rename_user_2@edx.org',
+            password='foo'
+        )
+
+        new_username = 'targetName'
+        not_found_username = 'notFoundName'
+
+        # ensures that the test database doesn't have a user with username `notFoundName`
+        self.assertEqual(0, len(User.objects.filter(
```

Comment:
Can you run Pylint and PEP8 against this file and address any issues in lines you've changed? 

I think the continuation here is invalid and should, instead, be along the lines of:

```python
self.assertEqual(
    0,
    len(User.objects.filter(
        username=username_nonexistent,
    )),
)
```